### PR TITLE
rsync upload, temp-dir flag warning

### PR DESCRIPTION
**Pull request by zepner**

### CODE REVIEW

File: source/docs/articles/local/rsync-and-sftp.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ $: export SITE=3ef6264e-51d9-43b9-a60b-6cc22c3129308as83
 $: rsync -rlvz --size-only --ipv4 --progress -e 'ssh -p 2222' $ENV.$SITE@appserver.$ENV.$SITE.drush.in:code/wp-content/uploads/index.php ~/Foo/sites/wp-content/uploads
 ```
 
+<div class="alert alert-danger" role="alert">
+<h4>Warning</h4>
+Remember to always use the `temp-dir flag` when using rsyc for uploads.</div>
```
Comment (ari-gold) [inline source/docs/articles/local/rsync-and-sftp.md]: typo. rsyc -> rsync

**Comment by rachelwhitton** [comment]
+1 thanks @zepner!

**Comment by bmackinney** [comment]
@davidstrauss can you review?

**Comment by rachelwhitton** [comment]
Verbal +1 from @bmackinney

**Pull request merged**